model/Task/DecryptDeliveryExecutionTask.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace oat\taoEncryption\Task;
+
+use oat\oatbox\extension\AbstractAction;
+use oat\taoEncryption\Service\Result\DecryptResultService;
+
+class DecryptDeliveryExecutionTask extends AbstractAction
```

Comment:
Why to copy/paste this class? It creates lot of duplicate and only the array keys change

Reply:
Those are all commits from the past ;)

I'm just trying to align the [_latest_ known-to-work extension version](https://github.com/oat-sa/extension-tao-encryption/releases/tag/v3.2.0.1) with the current release branch. That "backport" was _never_ merged into `develop`.

See [this comparison](https://github.com/oat-sa/extension-tao-encryption/compare/v3.2.0...v3.2.0.1) for reference (spoiler, it's exactly the contents of this PR).

Reply:
Perhaps, that port is not even needed then. I'm just aligning the version that was actually used with the current release.